python/sglang/srt/layers/amx_utils.py

```diff
@@ -28,6 +31,106 @@ def dim_is_supported(weight):
     return OC % TILE_N == 0 and IC % TILE_K == 0
 
 
+def _autoawq_to_int4pack(
+    qweight: torch.Tensor, qzeros: torch.Tensor, scales: torch.Tensor, is_w4a8: bool
+):
+    """Convert AutoAWQ weight format to sgl-kernel's CPU int4
+
+    Args:
+        qweight: (*, K, N / 8), int32
+        qzeros: (*, K / group_size, N / 8), int32
+        scales: (*, K / group_size, N), bfloat16
```

Comment:
you can :

* overload `convert_weight_packed`
* new api maybe `convert_weight_packed_scale_zp`
* simply skip packing scale and zp, use prefetch can also achieve good  performance and much simpler.

Reply:
Sure, will refine the pack function with `convert_weight_packed_scale_zp`